server/routes/api.js

```diff
@@ -11,4 +12,26 @@ router.route('/')
     res.status(201).send({ data: 'Posted!' });
   });
 
+router.route('/test')
```

Comment:
I get that you are just testing your DB code here, but you should not include anything in your code submissions that is development code not adding to the product.